Makefile

```diff
@@ -27,6 +28,18 @@ style:
 format:
 	black --line-length=99 $(PYFILES)
 
+## bandage:     unpack Bandage to enable assembly graphs visualizations
+bandage:
+	@if [[ "$(OS)" == "Linux" ]]; then \
+		yes A | unzip yeat/data/Bandage_Ubuntu_dynamic_v0_8_1.zip -d ~/Bandage; \
+		sudo apt install build-essential git libgl1-mesa-dev libxcb-xinerama0; \
+		sudo apt-get install libqt5gui5; \
+	elif [[ "$(OS)" == "Darwin" ]]; then \
+		yes A | unzip yeat/data/Bandage_Mac_v0_8_1.zip -d ~/Bandage; \
+	else \
+		echo "yeat does not support Bandage for: $(OS)"; \
+	fi
```

Comment:
I'd recommend unzipping into the conda environment rather than `~/Bandage/`. For macOS, you may even consider symlinking the Bandage executable to `$ENVDIR/bin/Bandage`.

Reply:
Also, unless I'm missing something, the backslash and semi-colon characters are redundant. The backslash is to allow a single command to span multiple lines (it literally escapes the newline character), whereas the semicolon marks the end of one command and the beginning of another. Unless you have a good reason to, I'd recommend cleaning that up a bit.

Reply:
Also also, apt/apt-get is Ubuntu specific. That's likely going to be the majority of Linux users, but not all. We should give priority support to CentOS and RedHat. If that requires listing OS-level dependencies not easily installed by conda, then I suggest we move that to installation instructions rather than handling every OS flavor in the Makefile.

Reply:
> For macOS, you may even consider symlinking the Bandage executable to $ENVDIR/bin/Bandage.

Tried to symlink; however, the binary file depends on all of the other files inside of the prepackaged directory. As a result, Bandage crashes.

Dropped the `make bandage` command. Making users update their $PATH accordingly to enable the Bandage command for their OS.